DeviceDetector.NET/DeviceDetector.NET.csproj

```diff
@@ -29,6 +29,7 @@
   <ItemGroup>
     <PackageReference Include="LiteDB" Version="4.1.4" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="NLog" Version="4.7.10" />
```

Comment:
Is there built in logging framework to avoid NLOG dependency? Maybe won't work due to multiple targets

Reply:
Just found the extensions Nuget which extends NLog to include Microsoft....ILogger, will use that in SelfService.

Reply:
Microsofts ILogger does require .Net 4.6.1 which shouldn't be a big problem.